docs/tutorials/solar_flares.ipynb

```diff
@@ -1132,7 +1132,8 @@
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3"
+   "pygments_lexer": "ipython3",
+   "version": "3.9.15"
```

Comment:
Unrelated to this PR: can you figure out why stripping the version does not seem to be working in the CI?

Reply:
Maybe we are simply missing some config options for `nbstripout`? https://github.com/kynan/nbstripout#stripping-metadata